airqo-platform/AirQo-frontend
# [NetManager] Track user preferences

#### Summary of Changes (What does this PR do?)
- Track user preferences such as pagination choices for the various tables

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Install requirements `npm install`
* Start application `npm run stage-mac` (macOS) or `npm run stage-pc` (windows Platform)
* Set pagination on (devices, locations, users, candidates) pages and refresh the respective page to ensure the pagination choice is persisted.

#### What are the relevant tickets?
- [PLAT-356](https://airqoteam.atlassian.net/browse/PLAT-356)

#### Screenshots (optional)
